friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './friends.css';
+
+
+class FriendForm extends React.Component {
+    constructor () {
+        super(); 
+        this.state = {
+            friend: {
+            name: '',
+            email: '',
+            age: '',
+            }
+        }
+    }
+
+    changeHandler = e => {
+        e.persist();
```

Comment:
don't need to use persist() here if only an object instead of a function is being passed into setState